src/ontology/cl-edit.owl

```diff
@@ -27533,6 +27515,16 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 <https://identifiers.org/ensembl:ENSMUSG00000005716>))
 
+# Class: obo:CL_4023038 (L6b glutamatergic cortical neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.34312") Annotation(oboInOwl:hasDbXref "PMID:30382198") obo:IAO_0000115 obo:CL_4023038 "Glutamatergic neuron located in cortical layer 6b. They transcriptomically related to corticothalamic-projecting neurons but have differential projections to the thalamus or anterior cingulate.")
```

Comment:
Small changes: "A glutamatergic neuron located in cortical layer 6b. They are transcriptomically related to corticothalamic-projecting neurons but have differential projections to the thalamus or anterior cingulate."